server/graphql/resolvers/article/queries.js

```diff
@@ -0,0 +1,94 @@
+const articleQueries = {
+  articles: (_, { search }) => {
+    console.log(search);
+    const articles = [
+      {
+        id: 1,
+        number: 1,
+        content:
+          'Esta prohibido manejar en las siguientes condiciones:\n\t 1. Teniendo los faros rotos\n\t 2. Teniendo las llantas ponchadas\nCualquiera de estos casos no será tolerado!',
+        keywords: [
```

Comment:
I wouldn't include this just yet, until we define it.

Reply:
Yeah, these are just mock up responses to design the webapp. The idea is that this is the generated result that should be returned and displayed to the client.